docs/concepts/jobs-overview.md

```diff
@@ -0,0 +1,59 @@
+(jobs-overview)=
+# Jobs Overview
+
+A `Job` is simply something that you would like to run once to completion. For example, running a simple Python script can be similar to a `Job` in that the script runs once to completion and then returns. In an ML workflow, sometimes running a simple Python script is not enough and additional functionality is required. For example giving cluster resources, specifying dependencies and a Docker image, integrating with Git repo, etc. 
+
+### Create a Job
+
+MLRun has the ability to add all of the above features and more when running a `Job`. To showcase this, we will create a job to run the code below which resides in a file titled `code.py`:
+
+```python
+def hello(context):
+    print("You just ran a job!")
+```
+
+To create the Job, use the `code_to_function` syntax and specify the `kind` like below:
+
+```python
+import mlrun
+
+job = mlrun.code_to_function(
+    name="my-job",                # Name of the job (displayed in console and UI)
+    filename="code.py",           # Python file or Jupyter notebook to run
+    kind="job",                   # Run as a job
+    image="mlrun/mlrun",          # Use this Docker image
+    handler="hello"               # Execute the function hello() within code.py
+)
+```
+
+You can read more about the `code_to_function` syntax **HERE**.
+
+### Run a Job Locally
+
+When prototyping, it is often useful to test the `Job` locally on your laptop or Jupyter environment before running on the larger cluster. This lets you ensure the job does what you want without using cluster resources.
+
+To do this, run the job and specify the `local=True` flag 
+
+```python
+run = job.run(local=True)
+```
+
+### Run a Job on the Cluster
+
+Finally, you can execute your job using cluster resources. This is usually the end goal when creating a job becuase it gives you much more flexibility into the configuration of the job.
```

Comment:
Finally, you can execute your job using cluster resources. This is usually the end goal when creating a job because it gives you much more flexibility into the configuration of the job.